torchci/rockset/commons/__sql/flaky_tests_across_jobs.sql

```diff
@@ -0,0 +1,102 @@
+with failed_jobs as (
+    SELECT
+        FIRST_VALUE(job.conclusion) OVER(
+            PARTITION BY CONCAT(w.name, ' / ', job.name)
+            ORDER BY
+                push.head_commit.timestamp ROWS BETWEEN 1 PRECEDING
+                AND 1 FOLLOWING
+        ) = 'success'
+        and NTH_VALUE(job.conclusion, 2) OVER(
+            PARTITION BY CONCAT(w.name, ' / ', job.name)
+            ORDER BY
+                push.head_commit.timestamp ROWS BETWEEN 1 PRECEDING
+                AND 1 FOLLOWING
+        ) = 'failure'
+        and LAST_VALUE(job.conclusion) OVER(
+            PARTITION BY CONCAT(w.name, ' / ', job.name)
+            ORDER BY
+                push.head_commit.timestamp ROWS BETWEEN 1 PRECEDING
+                AND 1 FOLLOWING
+        ) = 'success' as flaky,
+        job.id,
+        job.head_sha,
+        job.name as jobname,
+        w.id as workflow_id,
+        w.head_branch,
+        w.name as workflow_name,
+        w.run_attempt as workflow_run_attempt,
+        job._event_time as event_time
+    from
+        commons.workflow_job job
+        join commons.workflow_run w on w.id = job.run_id
+        and w.head_repository.full_name = 'pytorch/pytorch'
+        join push on push.head_commit.id = w.head_commit.id
+    where
+        job._event_time >= CURRENT_DATE() - HOURS(:numHours)
+        and w.head_branch = 'master'
+        and w.name in ('trunk', 'periodic', 'pull')
+        and job.name not like '%rerun_disabled_tests%'
+        and job.name not like '%mem_leak_check%'
```

Comment:
Hmm, this won't work for periodic as expected I think.  All periodic jobs now have `mem_leak_check` suffix IIRC

Reply:
after discussion, changed to exclude periodic entirely due to the higher probably of going from green -> red -> green but not being flaky